discussion/app/views/fragments/recommendationTooltip.scala.html

```diff
@@ -0,0 +1,16 @@
+<div class="tooltip-box tooltip-box--neutral tooltip-box--width4 tooltip-box-hidden js-rec-tooltip">
```

Comment:
Generally speaking, every DOM fragment inside a page that is hidden should have either the `hidden` or `aria-hidden="true"` attributes. All these `*-hidden` classes are only for people with sight 😭